### PR TITLE
feat(release) release process for the installer

### PR DESCRIPTION
**Pull request by kevinawoo**
- release process will be controlled by a Spinnaker pipeline
- the public install script uploaded to (https://s3-us-west-2.amazonaws.com/armory-web/install/release/kubernetes-installer/public-installer.sh) will get a dns name, suggestions on the name?

### CODE REVIEW

File: src/public-installer.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+cd "$(dirname "$0")"
+
+## This is the public installer script that will fetch the kubernetes installer run it
+## The intention is that the AWS install script pieces will one day be cherry-picked into here.
```
Comment (andrewbackes) [inline src/public-installer.sh]: Since this is public facing, we should take out these notes.

Comment (kevinawoo) [reply src/public-installer.sh]: kk

**Comment by kevinawoo** [comment]
Please don't merge yet! `envsubt` and `aws-cli` isn't available on jenkins. I'll need to run this in a container.

**Comment by kevinawoo** [comment]
@andrewbackes added the container runner: https://github.com/armory-io/k8s-installer/pull/151/commits/89e5f72c07e406bd729c9114a6d2b5d84f2efaa7

**Pull request merged**